src/test/java/org/iotexproject/antenna/rpcmethod/IoTeXGRPCTestInterface.java

```diff
@@ -1,5 +1,15 @@
 package org.iotexproject.antenna.rpcmethod;
 
+/**
```

Comment:
Can Java LICENSE comment is this style?

Reply:
What do you mean? This is like same java source file!